docs/data-model.md

```diff
@@ -0,0 +1,78 @@
+
+- `ontology_terms`:(id, curie, uri, label, status, description, cross_refs)
+
+The `uri` field represents the complete URI, e. g. http://www.ebi.ac.uk/efo/EFO_0000249, while `term` stores the corresponding CURIE (compact URI), in this case EFO:0000249. Note that while URL uses an underscore, CURIE uses a colon to separate namespace and an identifier. Both the URI and the CURIE can be obtained through the OLS query.
+A note to make, is that the `description` and `cross_refs` fields only apply to new term suggestions created via our web app. Other terms will have those fields empty.
+
+The ontology term status can be either `current`, `obsolete`, `deleted`, `needs_import`, `needs_creation`, `awaiting_import`, `awaiting_creation`. This status is automatically computed, except for moving out of the “awaiting_creation” status, which requires a manual check.
+
+- `mapping_suggestions` (id, trait_id [foreign_key], ontology_id [foreign_key], made_by[foreign_key], timestamp)
+
+- `reviews` (id, mapping_id [foreign_key], reviewer [foreign_key])
+
+- `comments` (id, trait_id [foreign_key], author[foreign_key], date, body)
+
+Comments apply to a trait in general, not to its specific state or the mappings. Essentially this is just free text displayed inline with the status and mapping changes.
+
+## Trait statuses and their transitions
+
+| Action                                                                                                                                                                       | Ontology term status | Mapping review status <br>(computed based<br> on reviews table) | Trait status - master<br>(computed based on <br>the previous two) |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------- | --------------------------------------------------------------- | ---------------------------------------------------------------- |
+| **Case 1. Curator chooses an existing term from EFO**                                                                                                                        | current              | not_reviewed                                                    | awaiting review                                                  |
+| A necessary number of reviews is accumulated                                                                                                                                 | current              | reviewed                                                        | current                                                          |
```

Comment:
Regarding the Markdown layout for this table (since you've raised concerns about it in the PR description): it's mostly fine as it is; this table is unlikely to be frequently changed, so the most important thing is that it should be readable in the final rendered Markdown, which it is.

However, if you want to improve readability of the source code as well, I can suggest not trying to keep column delimiters aligned. Markdown allows for this, and in this case it will look prettier. Compare, before:

```markdown
| A necessary number of reviews is accumulated                                                                                                                                 | current              | reviewed                                                        | current                                                          |
```

And after:

```markdown
| A necessary number of reviews is accumulated | current | reviewed | current |
```